main/main.cpp

```diff
@@ -852,7 +810,8 @@ void Main::test_cleanup() {
 
 	finalize_theme_db();
 
-	finalize_navigation_server();
+	NavigationServer2DManager::finalize_server(); // 2D needs to go first.
```

Comment:
Maybe a quick explanation on why 2D needs to go first ?

Reply:
There is no real 2D server backend which is why we need to init the 3D server before the 2D.
For finalize we go reverse with finalizing the 2D before 3D.